picard/ui/coverartbox.py

```diff
@@ -255,28 +283,29 @@ def load_remote_image(self, url, mime, data):
         try:
             coverartimage = CoverArtImage(
                 url=url.toString(),
+                types=[u'front'],
                 data=data
             )
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
         if isinstance(self.item, Album):
             album = self.item
-            album.metadata.append_image(coverartimage)
+            album.metadata.set_front_image(coverartimage)
```

Comment:
I implemented the radio buttons on the context menu not to clutter the options dialog with a new option, so this will be addressed by this branch:
https://github.com/antlarr/picard/compare/album-cover-art...antlarr:cover-art-context-menu?expand=1
You can see the result here: http://i.imgur.com/lyI9SNs.png
I'll create the pull request for the cover-art-context-menu branch as soon as this one is merged (the work there is already finished) since the commits in that branch depend on this PR.